presentations/AnalogL1/presentation.html

```diff
@@ -851,10 +851,10 @@
 - We need to consider the highest i<sub>L</sub> to be measured measured to ensure P<sub>is-loss</sub> &le; 50mW
  - Highest I<sub>L(RMS)</sub> is at lowest V<sub>L(RMS)</sub> and highest VA specified (i.e., I<sub>L(RMS)</sub> = 7.5VA/13.6V = 0.55A<sub>RMS</sub>)
 - Using this information we can derive the relation
-\\[ P\_{is-loss} = I^2\_{L(RMS)} R\_s \quad \Rightarrow \quad 0.05 \geq 0.55^2 R\_s \quad \Rightarrow \quad R\_s \leq 165 m\Omega \\]
+\\[ P\_\text{is-loss} = I^2\_\text{L(RMS)}\cdot R\_\text{s} \quad \Rightarrow \quad 0.05 \geq 0.55^2 R\_\text{s} \quad \Rightarrow \quad R\_\text{s} \leq 165\, \text{m}\Omega \\]
 - If R<sub>s</sub> is choosen as 160m&Omega;, V<sub>is</sub> measured at both highest and lowest i<sub>L</sub> are
```

Comment:
> **Warning**  
> This may cause merge conflicts with #4.

To resolve this conflict, accept the changes from this branch and manually apply the typo fix in #4 by changing `choosen` -> `chosen`.